labs/middleware.py

```diff
@@ -86,15 +89,27 @@ def check_refusal(llm_response):
     return False, ""
 
 
+def check_invalid_json_response(llm_response):
+    response_string = llm_response["choices"][0]["message"]["content"]
+    if not is_valid_json(response_string):
```

Comment:
We don't need to validate that the response is valid json.

https://platform.openai.com/docs/guides/structured-outputs/json-mode